tests/sentry/api/endpoints/test_organization_releases.py

```diff
@@ -492,6 +492,114 @@ def test_commits(self):
+                        "repository": "laurynsentry/helloworld",
+                        "author_email": "lauryndbrown@gmail.com",
+                        "timestamp": "2018-11-30T22:51:14+03:00",
+                        "author_name": "Lauryn Brown",
+                        "message": "Changed release",
+                        "id": "be2fe070f6d1b8a572b67defc87af2582f9b0d78"
+                    }
+                ]
+            }
+        )
+
+        assert response.status_code == 201, (response.status_code, response.content)
+        assert response.data['version']
+
+        release = Release.objects.get(
+            organization_id=org.id,
+            version=response.data['version'],
+        )
+
+        repo = Repository.objects.get(
+            organization_id=org.id,
+            name='laurynsentry/helloworld',
+        )
+        assert repo.provider is None
+
+        rc_list = list(
+            ReleaseCommit.objects.filter(
+                release=release,
+            ).select_related('commit', 'commit__author').order_by('order')
+        )
+        assert len(rc_list) == 2
+        for rc in rc_list:
+            assert rc.organization_id
+
+        author = CommitAuthor.objects.get(
+            organization_id=org.id,
+            email='lauryndbrown@gmail.com'
+        )
+        assert author.name == 'Lauryn Brown'
+
+        commits = [rc.commit for rc in rc_list]
+        commits.sort(key=lambda c: c.date_added)
+
+        assert commits[0].organization_id == org.id
+        assert commits[0].repository_id == repo.id
+        assert commits[0].key == '2d1ab93fe4bb42db80890f01f8358fc9f8fbff3b'
+        assert commits[0].author_id == author.id
+        assert commits[0].message == 'made changes to hello.'
+
+        assert commits[1].organization_id == org.id
+        assert commits[1].repository_id == repo.id
+        assert commits[1].key == 'be2fe070f6d1b8a572b67defc87af2582f9b0d78'
+        assert commits[1].author_id == author.id
+        assert commits[1].message == 'Changed release'
+
+        file_changes = CommitFileChange.objects.filter(
+            organization_id=org.id
+        ).order_by('filename')
+
+        file_change = file_changes[0]
+        assert file_change.type == 'M'
+        assert file_change.filename == 'hello.py'
+        assert file_change.commit_id == commits[0].id
+
+        file_change = file_changes[1]
+        assert file_change.type == 'A'
+        assert file_change.filename == 'templates/goodbye.html'
+        assert file_change.commit_id == commits[1].id
+
+        file_change = file_changes[2]
+        assert file_change.type == 'M'
+        assert file_change.filename == 'templates/hello.html'
+        assert file_change.commit_id == commits[1].id
+
+        file_change = file_changes[3]
+        assert file_change.type == 'D'
+        assert file_change.filename == 'templates/hola.html'
+        assert file_change.commit_id == commits[0].id
```

Comment:
You added custom assertions for release project tests. It would be nice to resuse those here.